patients/get_patient_info.php

```diff
@@ -71,112 +90,112 @@
           ." FROM ".TBL_PAT_ACC_ON
           ." WHERE ".COL_POR_USER."=?";
             $auth = privQuery($sql, array($_POST['uname']));
-    if($auth===false)
+if($auth===false)
+{
+    session_destroy();
+    header('Location: '.$landingpage.'&w');
+    exit;
```

Comment:
sorry about that. my brain was playing tricks on me. I should of done a scan through this ignoring whitespacing before reporting this:
https://github.com/openemr/openemr/pull/925/files?w=1
(note that this made it clear to me that the only non whitespace changes are the header)